README.md

```diff
@@ -101,7 +102,7 @@ to using a triton-origin image.
 
 Ultimately the authority for which sdcnode builds and triton-origin images are
 available is defined in this repo and [sdcnode
-nodeconfigs.json](https://github.com/joyent/sdcnode/blob/master/nodeconfigs.json)
+nodeconfigs.json](https://github.com/TritonDataCenter/sdcnode/blob/master/nodeconfigs.json)
 files. We will try to keep this table up to date:
 
 | triton-origin image            | based on                     | sdcnode compatible build         | `NODE_PREBUILT_VERSION`              |
```

Comment:
Can you add 21.4.0 and 22.4.0 here?

Reply:
I added 21.4.0, 22.4.0 will have to wait until chicken-and-egg is complete.